package.json

```diff
@@ -121,6 +121,9 @@
         "node": ">=10.x",
         "yarn": ">=1.10.0"
     },
+    "dependencies": {
```

Comment:
probably move this to dev deps and peer deps.

Reply:
This is a real dependency, though. If this dependency is not satisfied, the code will not run properly and the unit tests will not work....

Reply:
Dev deps are also real dependencies. And with peer deps, the peer needs to install it also.

Reply:
moved to peer dependencies now